app/assets/stylesheets/components/flash.scss

```diff
@@ -1,9 +1,12 @@
+@import "govuk-frontend-rails";
```

Comment:
Technically not, but it documents the dependency, since this file uses the Sass variables set by `govuk-frontend-rails`, and doesn't result in it being imported more than once.

Reply:
Makes sense. In that case should we add it to `a11y_dialog.scss` and `currency_input.scss`, they both use GOV.UK mixins?

Reply:
Done. Good shout. I assumed it was only `flash.scss` because that's the one it was deleted from.